test_util/run-docker

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -x
+set -o errexit -o pipefail
+env
+
+function finish {
+  if [ -d dcos-docker ]; then
+    sudo make -C dcos-docker clean
+  fi
+}
+trap finish EXIT
+
+if [ ! -d dcos-docker ]; then
+  git clone https://github.com/dcos/dcos-docker
+else
+  git -C dcos-docker fetch origin -t +refs/heads/*:refs/remotes/origin/*
+fi
+
+curl -v $INSTALLER_URL > dcos-docker/dcos_generate_config.sh
+
+set +o errexit
+
+finish
+make -C dcos-docker
+make -C dcos-docker test
```

Comment:
Do we need to pass env variables here to configure `MASTER_COUNT` and `AGENT_COUNT` and INSTALLER_URL` as specified in the #417?